docs/reference/filters.md

```diff
@@ -3352,6 +3352,15 @@ tracingTag("http.flow_id", "${request.header.X-Flow-Id}")
 
 This filter works just like [tracingTag](#tracingtag), but is applied after the request was processed. In particular, [template placeholders](#template-placeholders) referencing the response can be used in the parameters.
 
+### tracingTagFromResponseIfStatus
+
+Example: set Error tag to true in case status code is >=500
+
+```
+tracingTagFromResponseIfStatus("Error", "true", ">499)
+```
```

Comment:
How can I set a tag for all 4xx responses?
Maybe instead of micro syntax we could ask users to specify min and max status values:
```
tracingTagFromResponseIfStatus("Error", "true", 400, 499)
```

Reply:
yeah, makes also sense. The question is also if I should do more this:
```
tracingErrorTagFromResponseIfStatus(400,499)
```

Or if the key-value setting makes sense.

Reply:
I think key-value is good, for generic thing we can implement this https://github.com/zalando/skipper/pull/3202#discussion_r1725088249 later